tools/accuracy_checker/configs/colorization-v2-norebal.yml

```diff
@@ -0,0 +1,37 @@
+evaluations:
+  - name: colorization-v2-norebal
+    module: custom_evaluators.colorization_v2_evaluator.ColorizationEvaluator
+    module_config:
+        network_info:
+            test:
+                model:   public/colorization-v2-norebal/FP32/colorization-v2-norebal.xml
+                weights: public/colorization-v2-norebal/FP32/colorization-v2-norebal.bin
+                color_reconstruction:
+                -   coeff: public/colorization-v2/colorization-v2.npy
```

Comment:
do you test relative path for this parameter? I am not sure that it will work without config_reader modification....

Reply:
@eaidova corrected)